vtkParallelTransportFrame.h

```diff
@@ -26,10 +26,12 @@
 ///
 /// References:
 /// - Parallel transport theory: R. Bishop, "There is more than one way to frame a curve",
-///   American Mathematical Monthly, vol. 82, no. 3, pp. 246�251, 1975
+///   American Mathematical Monthly, vol. 82, no. 3, pp. 246�251, 1975
 /// - Parallel transport implementation: Piccinelli M, Veneziani A, Steinman DA, Remuzzi A, Antiga L.
 ///   "A framework for geometric analysis of vascular structures: application to cerebral aneurysms.",
 ///   IEEE Trans Med Imaging. 2009 Aug;28(8):1141-55. doi: 10.1109/TMI.2009.2021652.
+/// - Wang, W., J ̈uttler, B., Zheng, D., and Liu, Y. 2008. Computation of rotation minimizing frame. ACM Trans. Graph. 27, 1, Article 2 (March 2008), 18 pages.
+///   DOI = 10.1145/1330511.1330513 http://doi.acm.org/10.1145/1330511.1330513
```

Comment:
Add open-access URL as well (https://www.microsoft.com/en-us/research/wp-content/uploads/2016/12/Computation-of-rotation-minimizing-frames.pdf)